zombienet_tests/register-para.js

```diff
@@ -0,0 +1,10 @@
+async function run(nodeName, networkInfo, args) {
+    const paraIdStr = args[0];
+    const para = networkInfo.paras[paraIdStr];
+    const relayNode = networkInfo.relay[0];
+
+    console.log(para);
+    await zombie.registerParachain(parseInt(paraIdStr,10),para.wasmPath, para.statePath, relayNode.wsUri, "//Alice", true);
```

Comment:
@ordian / @skunert this needs the last version of zombienet. I will change it in this branch.
Thanks!